nvtabular/framework_utils/torch/models.py

```diff
@@ -107,4 +114,4 @@ def forward(self, x_cat, x_cont):
         if self.max_output:
             x = self.max_output * torch.sigmoid(x)
         x = x.view(-1)
-        return x
+        return {"predictions": x}
```

Comment:
Does the output have to change here? Looks like the places where predictions happen are also updated to fetch `predictions`, so wondering if this change washes out and can be dropped

Reply:
so actually when you remove this you get the following error from triton: `Failed to process the request(s) for model 'test_name_pt', message: ValueError: output of the forward function should be a dict` this is why I added that change it needs to be a dictionary on the way out for triton to accept it.

Reply:
Makes sense!